en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/using-geo-location-based-statistics/creating-geo-location-data-set.md

```diff
@@ -1,6 +1,6 @@
-# Updating Geo Location Data Set
+# Creating Geo Location Data Set
 
-Follow the procedure below in order to update your existing Geo Location dataset to use in [Configuring Geolocation Based Statistics](../configuring-geolocation-based-statistics/).
+Follow the procedure below in order to create a Geo Location dataset to use when [Configuring Geolocation Based Statistics](../configuring-geolocation-based-statistics/).
```

Comment:
Check this link. It should start with {{base_path}} or if u are linking to the same page it will start with #

Reply:
Fixed in d9c29843ef4201cb9eca588053bea4aaecd174e0